.circleci/config.yml

```diff
@@ -30,35 +30,35 @@ commands:
           key: source-v3-{{ .Branch }}-{{ .Revision }}
           paths:
             - ".git"
-  yarn_install:
+  npm_install:
     description: "Install node modules"
     steps:
-      - restore_cache:
-          name: Restoring node_modules
-          keys:
-            - source-v3-modules-{{ checksum "yarn.lock" }}
-            - source-v3-modules
+      # - restore_cache:
+      #     name: Restoring node_modules
+      #     keys:
+      #       - source-v3-modules-{{ checksum "package-lock.json" }} CircleCI retention cache for 15 days
```

Comment:
make sure you remove "CircleCI retention cache for 15 days" in future. that's just a explanation

Reply:
@shudarshon-binary, for sure. I'll remove the text in next PR